### DISCUSSION TITLE
[USER ERROR] Second time solving kata not always showing up

### DISCUSSION THREAD

**Discussion by Nik-Sharp** [codewars/codewars.com#2551] [Q&A] [answered]
So sometimes when solving a Kata that I've done before, the new solution will not appear in the "My Solutions" tab. However, sometimes it will. Is there a reason for this? Thanks!

**Comment by Nik-Sharp** [comment, marked as answer]
Since I don't know how to delete a post, would like to mention that it just didn't appear immediately, so user error. Thanks!

**Reply by hobovsky** [reply]
Just to be clear, there is a bug which can prevent your solution from showing up in Solutions tab, so the problem can, and probably will, reappear: https://github.com/codewars/codewars.com/issues/1884#issuecomment-805349522
